examples/requirements/docstring.py

```diff
@@ -0,0 +1,102 @@
+
+from avocado import Test
+from avocado.utils import vmimage
+
+
+class GPL2(Test):
+    """
+    Example test using docstring requirements.
+
+    :avocado: requirement={"type": "file", "name": "gpl-2.0.txt", "locations": "https://mirrors.kernel.org/gnu/Licenses/gpl-2.0.txt", "asset_hash": "4cc77b90af91e615a64ae04893fdffa7939db84c"}
```

Comment:
Hi @willianrampazzo, just out of curiosity:

Is this trigger some linter alerts (80 columns)? I mean, is the parsing able to handle multiple lines?

Reply:
> Is this trigger some linter alerts (80 columns)? I mean, is the parsing able to handle multiple lines?

Yes, the line triggers an alert on linters. I did a test here and the parser is not able to handle multiple lines the way it is now.